src/datum_api.c

```diff
@@ -89,10 +89,22 @@ static void html_leading_zeros(char * const buffer, const size_t buffer_size, co
 }
 
 void datum_api_var_DATUM_SHARES_ACCEPTED(char *buffer, size_t buffer_size, const T_DATUM_API_DASH_VARS *vardata) {
-	snprintf(buffer, buffer_size, "%llu  (%llu diff)", (unsigned long long)datum_accepted_share_count, (unsigned long long)datum_accepted_share_diff);
+	// Show pool shares when in pooled mode, stratum client shares when in solo mode
+	// This allows solo miners to see their actual mining activity instead of zeros
+	if (datum_config.datum_pooled_mining_only) {
+		snprintf(buffer, buffer_size, "%llu  (%llu diff)", (unsigned long long)datum_accepted_share_count, (unsigned long long)datum_accepted_share_diff);
+	} else {
+		snprintf(buffer, buffer_size, "%llu  (%llu diff)", (unsigned long long)stratum_client_accepted_share_count, (unsigned long long)stratum_client_accepted_share_diff);
```

Comment:
Why not reuse the existing `datum_accepted_share_count` and `datum_accepted_share_diff` fields?